docs/guides/git-vs-composer-install.md

```diff
@@ -0,0 +1,72 @@
+#Git vs Composer Installation of Magento with MFTF
+
+
+###How to download Magento + MFTF from GitHub?
+
+If you are planning on contributing a PR to the Magento 2 codebase, you can download Magento 2 from GitHub. Contribution to the codebase is done using the 'fork and pull' model where contributors maintain their own fork of the repo. This repo is then used to submit a pull request to the base repo.
+
+Install guide: [GitHub Installation](https://devdocs.magento.com/mftf/docs/getting-started.html)
+
+###How to download Magento + MFTF via Composer?
+
+Composer install downloads released packages of Magento 2 from the composer repo [https://repo.magento.com](https://repo.magento.com).
+
+All Magento modules and their MFTF tests are put under `<vendor>` directory for convenience of 3rd party developers. With this setup, you can keep your custom modules separate from the core. You can also develop modules in a separate VCS repository and add them to your `composer.json` which will allow them to be installed into the `vendor` directory.
+
+Install guide: [Composer based Installation](https://devdocs.magento.com/guides/v2.3/install-gde/composer.html)
+
+
+###Managing MFTF test artifacts - Composer vs GitHub
```

Comment:
To me this section talks more about benefits of using one way or the other. Maybe the title can be more specific for that?

Reply:
Made this generic to Managing modules - Composer vs Github.